DuckDuckGoTests/YoutublePlayerNavigationHandlerTests.swift

```diff
@@ -582,4 +582,24 @@ class DuckPlayerNavigationHandlerTests: XCTestCase {
         XCTAssertNil(tabNavigator.openedURL, "No new tabs should open")
     }
     
+    @MainActor
```

Comment:
Is @MainActor needed here? I think that Bartek is working thorwards parallelising the unit tests and this could interfeer, In any case I don't think we should run tests on main therad.

Reply:
Good call.  Although not required, it was to align with the convention that any code interacting with WebKit or UI components runs on the main thread.   

Like If you replace MockWebView with a real WKWebView or integrate other WKNavigationDelegate-related logic in the future, @MainActor will help avoid threading issues.   Not likely happen in a test though.   I'll make a note to revise/update the class.